pango-view/viewer-render.h

```diff
@@ -67,14 +67,22 @@ extern gboolean opt_rtl;
 extern double opt_rotate;
 extern gboolean opt_auto_dir;
 extern const char *opt_text;
+extern const char *opt_text_file;
 extern gboolean opt_waterfall;
+extern gboolean opt_trim;
+extern double opt_trim_width;
+extern double opt_trim_height;
+extern double opt_trim_x;
+extern double opt_trim_y;
```

Comment:
May be use `PangoRegtangle` instead of the 4 variables, and name it `layout_extents` or something since it is not really an option.

Reply:
Fixed, although I had to introduce a new struct in order to accept double values.